engine.c

```diff
@@ -1043,7 +1043,7 @@ cmd_keyboard(int ch)
 		c->exec();
 		break;
 	}
-	case KEY_BACKSPACE:
+	case 0x7f:
```

Comment:
```suggestion
	case KEY_BACKSPACE:
	case 0x7f:
```

Can you put `case KEY_BACKSPACE` back above this line, so both, KEY_BACKSPACE and 0x7f are in there?